app.css

```diff
@@ -1,7 +1,58 @@
 /* apply a natural box layout model to all elements, but allowing components to change */
 html {
+  display: block;
   box-sizing: border-box;
+  font-size: 16px;
+  font-family: sans-serif;
 }
 *, *:before, *:after {
   box-sizing: inherit;
 }
+/* General */
+
+body {
+  max-width: 1000px;
+  margin: auto;
+  padding: inherit;
+}
+
+/* Image Styling */
+
+.column {
+  float: left;
+  display: inline-block;
+  margin: 5px 5px 0 5px;
+  padding: 0;
+}
+
+.octocat-pics {
+  float: left;
+  display: inline-block;
+  width: 300px;
```

Comment:
This width should probably be set on the `.column` and then use percents to make the picture fill the column.